### DISCUSSION TITLE
How can I display the properties metadata on the hover preview?

### DISCUSSION THREAD

**Discussion by dresgonzalez13** [Richardsl/heatmap-calendar-obsidian#104] [Q&A]
I see in the GIF provided on the READ ME page, that you're able to preview not only the content of the note but also its properties.
<img width="457" alt="Screenshot 2023-12-05 at 11 50 29 AM" src="https://github.com/Richardsl/heatmap-calendar-obsidian/assets/106787235/0d79ab3e-2e03-47b6-8163-f96b8909c3e0">

**Comment by belaset** [comment]
I think that's what the Metatable plugin was for, but unfortunately it's been deprecated :(

**Comment by discipleofzen** [comment]
Well... That was awesome. I only came to notice this today, but I miss it already. ;) Hope something similar will come up again...

**Comment by SebSwan** [comment]
Any news about this features ? Thanks

**Comment by occultliquidity** [comment]
To do this, download the plugin called Hover editor by nothingislost. 

![Screenshot 2024-12-14 at 15 31 07](https://github.com/user-attachments/assets/e99d15ce-0fe5-4987-91d3-0f8eb0b91652)

As you can see my example, when I hover I see all the front matter. 


https://github.com/user-attachments/assets/acccd95d-5be1-4487-8bd2-7a9b942d6d05
